bot/exts/info/information.py

```diff
@@ -243,7 +243,7 @@ async def create_user_embed(self, ctx: Context, user: FetchedMember) -> Embed:
 
         if on_server:
             joined = discord_timestamp(user.joined_at, TimestampFormats.RELATIVE)
-            roles = ", ".join(role.mention for role in user.roles[1:])
+            roles = ", ".join(role.mention for role in user.roles[:0:-1])
```

Comment:
```suggestion
            roles = ", ".join(role.mention for role in reversed(user.roles))
```
I find this easier to read, so it's clear what this is doing to people coming across this code in future.

This also has the benefit of just being a generator, rather than making a whole new list

Reply:
I agree, but we'll still need a slice for removing the `@everyone` role, right? That's why the `[1:]` was there and that's why I didn't do `[::-1]`, but I did `[:0:-1]`

Reply:
If we need a slice, THEN a reversed iterator, I believe everything in a slice is better.

Reply:
Perhaps this can be resolved with a comment explaining the slice, so we get the best of both worlds?

Reply:
Ah right, yea that would make sense why the :0 is there then. Could you add a comment to explain that? :D 

Seeing [:0:-1] just looked wrong to me, but it makes sense now. So I think a comment could really help the next person that glances at this and may think the same

Reply:
Got it, comment added, resolved!